test/unit/deletion_test.rb

```diff
@@ -51,6 +51,15 @@ class DeletionTest < ActiveSupport::TestCase
     assert_equal deletion.rubygem, @version.rubygem.name
   end
 
+  test "expire API memcached" do
+    Rails.cache.write("deps/v1/#{@version.rubygem.name}", "omg!")
+    assert !nil, Rails.cache.fetch("deps/v1/#{@version.rubygem.name}")
```

Comment:
I was trying to show the cache value goes from not-nil to nil, when we call `delete_gem` -- is it ok if I just simplify this to `refute_nil`?